src/js/visual/configurator-csr.js

```diff
@@ -1162,6 +1168,8 @@ ripe.ConfiguratorCsr.prototype._deinitCsr = function() {
     this._destroyInitialsResources();
     this._destroyScene();
 
+    this._unregisterConfigGlobalHandlers();
+

```

Comment:
Move this to `deinit()` here: https://github.com/ripe-tech/ripe-sdk/blob/563ba2394316f4ce2fdb6ecbed7f818afa246997/src/js/visual/configurator-csr.js#L157

As the registering is done in the `init()` thus the unregister should be done in it's corresponding `deinit()`